### PR TITLE
Feature integration

### PR DESCRIPTION
**Pull request by Abhishek-Raj-AT**
Summary:
Add integration in features of the page
Change log:
Added integration in features of the page

Test case:
- [ ]  Check API integration in Films
- [ ] Check API Integration in Starship
- [ ] Check API Integration in People
- [ ] Check API Integration in Vehicle
- [ ] Check API Integration in species
- [ ] Check API Integration in Planets
- [ ] Please check the functionality 
- [ ] Visit the page then by default go to the Home page
- [ ] When clicking on the view link then go to a single page and show all details of this link

Testing Url:
https://star-wars-character-4s2qmnysg-abhishek-raj-at.vercel.app/

### CODE REVIEW

File: src/pages/Films/index.tsx [hunk 1]
```diff
@@ -1,6 +1,36 @@
...
-    return (
-        <h1>film here</h1>
-    )
-}
-export default Films
\ No newline at end of file
+import { useSelector } from "react-redux";
+import { IRootState, useAppDispatch } from "../../redux/store";
+import { useEffect } from "react";
+import { getFilmActions } from "../../redux/FilmSlice/FilmAyscThunk";
```
Comment (bhavikg-dev) [inline src/pages/Films/index.tsx]: Is this name `FilmAyscThunk ` proper ?

File: src/pages/People/index.tsx [hunk 2]
```diff
@@ -1,6 +1,35 @@
...
-    return(
-        <>people here</>
-    )
-}
-export default People
\ No newline at end of file
+import { useSelector } from "react-redux";
+import { IRootState, useAppDispatch } from "../../redux/store";
+import { useEffect } from "react";
+import { getPeopleActions } from "../../redux/PeopleSlice/PeopleAyscThunk";
```
Comment (bhavikg-dev) [inline src/pages/People/index.tsx]: Same as above

File: src/pages/Planets/index.tsx [hunk 3]
```diff
@@ -1,6 +1,76 @@
...
-    return(
-        <h1>Planets here</h1>
-    )
-}
-export default Planets
\ No newline at end of file
+import { useSelector } from "react-redux";
+import { IRootState, useAppDispatch } from "../../redux/store";
+import { useEffect } from "react";
+import { getPlanetActions } from "../../redux/PlanetsSlice/PlanetAyscThunk";
```
Comment (bhavikg-dev) [inline src/pages/Planets/index.tsx]: Same as above

File: src/pages/Species/index.tsx [hunk 4]
```diff
@@ -1,8 +1,79 @@
...
-            Species name available
-        </h1>
-    )
-}
-export default Species
\ No newline at end of file
+import { useSelector } from "react-redux";
+import { IRootState, useAppDispatch } from "../../redux/store";
+import { useEffect } from "react";
+import { getSpeciesActions } from "../../redux/SpeciesSlice/SpeciesAyscThunk";
```
Comment (bhavikg-dev) [inline src/pages/Species/index.tsx]: Same as above

File: src/pages/StarShips/index.tsx [hunk 5]
```diff
@@ -1,6 +1,42 @@
...
-    return(
-        <h1> StarShip here</h1>
-    )
-}
-export default StarShips
\ No newline at end of file
+import { useSelector } from "react-redux";
+import { IRootState, useAppDispatch } from "../../redux/store";
+import { useEffect } from "react";
+import { getStarshipActions } from "../../redux/StarshipSlice/StarshipAyscThunk";
```
Comment (bhavikg-dev) [inline src/pages/StarShips/index.tsx]: Same as above

File: src/pages/Vehicle/index.tsx [hunk 6]
```diff
@@ -1,6 +1,84 @@
-const Vehicle =()=>{
-    return(
-        <h1> vehicle items here</h1>
-    )
-}
-export default Vehicle
\ No newline at end of file
+import { useSelector } from "react-redux";
+import { IRootState, useAppDispatch } from "../../redux/store";
+import { getVehicleActions } from "../../redux/VehicleSlice/VehicleAyscThnk";
```
Comment (bhavikg-dev) [inline src/pages/Vehicle/index.tsx]: Same as above

File: src/redux/FilmSlice/index.tsx [hunk 7]
```diff
@@ -1,3 +1,4 @@
+import { getFilmActions } from "./FilmAyscThunk";
```
Comment (bhavikg-dev) [inline src/redux/FilmSlice/index.tsx]: Same as above

File: src/redux/PlanetsSlice/index.tsx [hunk 8]
```diff
@@ -0,0 +1,32 @@
+import { createSlice } from "@reduxjs/toolkit";
+import { PlanetList } from "./PlanetTypes";
+import { getPlanetActions } from "./PlanetAyscThunk";
```
Comment (bhavikg-dev) [inline src/redux/PlanetsSlice/index.tsx]: Same as above

File: src/redux/SpeciesSlice/index.tsx [hunk 9]
```diff
@@ -0,0 +1,35 @@
+import { createSlice } from "@reduxjs/toolkit";
+import { getSpeciesActions } from "./SpeciesAyscThunk";
```
Comment (bhavikg-dev) [inline src/redux/SpeciesSlice/index.tsx]: Same as above

File: src/redux/StarshipSlice/index.tsx [hunk 10]
```diff
@@ -0,0 +1,35 @@
+import { createSlice } from "@reduxjs/toolkit";
+import { StarshipList } from "./StarshipTypes";
+import { getStarshipActions } from "./StarshipAyscThunk";
```
Comment (bhavikg-dev) [inline src/redux/StarshipSlice/index.tsx]: Same as above

File: src/redux/VehicleSlice/index.tsx [hunk 11]
```diff
@@ -0,0 +1,35 @@
+import { createSlice } from "@reduxjs/toolkit";
+import { VehicleList } from "./Vehicletypes";
+import { getVehicleActions } from "./VehicleAyscThnk";
```
Comment (bhavikg-dev) [inline src/redux/VehicleSlice/index.tsx]: Same as above

**Review by bhavikg-dev — CHANGES_REQUESTED** [review]

File: src/pages/Films/index.tsx (hunk 1 above)
Comment (Abhishek-Raj-AT) [reply src/pages/Films/index.tsx]: I solved this issue

**Review by bhavikg-dev — APPROVED** [review]

**Pull request merged**